src/sindri/sindri.py

```diff
@@ -353,6 +353,41 @@ def create_circuit(self, circuit_upload_path: str, wait: bool = True) -> str:
         # Circuit compilation success!
         return circuit_id
 
+    def delete_circuit(
+        self, circuit_id: str
+    ) -> tuple[int, dict | list]:
+        """
+        Delete a circuit by hitting the `/circuit/<circuit_id>/delete` endpoint.
+        This also deletes all of the circuit's proofs.
+
+        Return the `response.status_code` and the JSON decoded `response`.
+        This may raise `Sindri.APIError`.
+        This does not check if the `response.status_code` is the successful response (200).
+        """
+        response_status_code, response_json =  self._hit_api("DELETE", f"circuit/{circuit_id}/delete")
+        if response_status_code != 200:
+            raise Sindri.APIError(
+                f"Unable to delete circuit_id={circuit_id}."
+                f" status={response_status_code} response={response_json}"
+            )
+
+    def delete_proof(
+        self, proof_id: str
+    ) -> tuple[int, dict | list]:
+        """
+        Delete a proof by hitting the `/proof/<proof_id>/delete` endpoint.
+
+        Return the `response.status_code` and the JSON decoded `response`.
+        This may raise `Sindri.APIError`.
+        This does not check if the `response.status_code` is the successful response (200).
```

Comment:
It does though? Same comment for `delete_circuit` above.

Reply:
Updated docstring: [4b6f5ef](https://github.com/Sindri-Labs/sindri-python/pull/21/commits/4b6f5ef85b5d0a339d92107ad5e560f5e70656f2)